config/environments/production.rb

```diff
@@ -65,19 +65,7 @@
   # with SQLite, MySQL, and PostgreSQL)
   # config.active_record.auto_explain_threshold_in_seconds = 0.5
 
+  config.action_mailer.perform_deliveries = false  
```

Comment:
Did you mean to add this line into local.rb instead?
I would think we'd want this to be `true` here.